cekit/cache/cli.py

```diff
@@ -36,11 +36,12 @@ def ls():
 @click.option('--md5', metavar="CHECKSUM", help="The md5 checksum of the artifact.")
 @click.option('--sha1', metavar="CHECKSUM", help="The sha1 checksum of the artifact.")
 @click.option('--sha256', metavar="CHECKSUM", help="The sha256 checksum of the artifact.")
-def add(location, md5, sha1, sha256):  # pylint: disable=unused-argument
-    if not (md5 or sha1 or sha256):
+@click.option('--sha512', metavar="CHECKSUM", help="The sha512 checksum of the artifact.")
+def add(location, md5, sha1, sha256, sha512):  # pylint: disable=unused-argument
+    if not (md5 or sha1 or sha256, sha512):
```

Comment:
Here is the issue, there should be no comma, but `or` string.

Reply:
oops, moving too fast. good spot!